migrations/20230501-init-schema.sql

```diff
@@ -0,0 +1,49 @@
+CREATE DATABASE quiz_webapp;
+use quiz_webapp;
+
+CREATE TABLE subjects (
+	id int auto_increment primary key,
+	name varchar(120) NOT NULL,
+	image_url varchar(250) NOT NULL
+);
+
+CREATE TABLE quizzes (
+	id int auto_increment primary key,
+	subject_id int,
+	name varchar(100) NOT NULL,
+	foreign key (subject_id) references subjects(id)
+);
+
+CREATE TABLE links (
+	id int auto_increment primary key,
+	quiz_id int,
+	name varchar(150),
+	url varchar(500),
+	foreign key (quiz_id) references quizzes(id)
+);
+
+CREATE TABLE questions (
+	id int auto_increment primary key,
+	quiz_id int,
+	question_text varchar(200),
+	option_1 varchar(250),
+	option_2 varchar(250),
+	option_3 varchar(250),
+	answer varchar(250),
```

Comment:
[SHOULD] In Intert into method, answer seems like integer. You should use int or varchar(1) instead of varchar(250).